frontend/src/bundles/studio/store/slice.ts

```diff
@@ -0,0 +1,41 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { type PayloadAction } from '@reduxjs/toolkit';
+
+import { type Script } from '../types/studio.type.js';
+
+type State = {
+    scripts: Array<Script>;
+};
+
+const initialState: State = {
+    scripts: [],
+};
+
+const { reducer, actions, name } = createSlice({
+    initialState,
+    name: 'studio',
+    reducers: {
+        addScript(state, action: PayloadAction<string>) {
+            const script = {
+                id: self.crypto.randomUUID(),
```

Comment:
Does `self.crypto.randomUUID()` properly work? There could be some issues related to crypto in React

Reply:
Works fine for me. Also it was used in react homework starter